parse_vary_num_tasks.py

```diff
@@ -64,36 +64,31 @@ def main(argv):
     num_tasks = num_tasks_values[0]
 
     ideal_runtimes_millis = []
-    ideal_runtimes_utilization_millis = []
     ideal_map_runtimes_millis = []
     actual_map_runtimes_millis = []
     ideal_reduce_runtimes_millis = []
     actual_reduce_runtimes_millis = []
 
     for job in all_jobs:
       job_ideal_millis = 0
-      job_ideal_utilization_millis = 0
       for (stage_id, stage) in job.stages.iteritems():
-        job_ideal_millis += stage.ideal_time(cores_per_machine = num_cores, disks_per_machine = 0)
-        stage_ideal_utilization_millis = stage.ideal_time_utilization(num_cores)
-        job_ideal_utilization_millis += stage_ideal_utilization_millis
+        stage_ideal_millis = stage.ideal_time_s(num_cores_per_executor = num_cores)
```

Comment:
Good point -- fixed! This is exactly why I changed the name of the ideal time function, was so that these errors would be more obvious.